nixops/backends/ec2.py

```diff
@@ -430,23 +430,27 @@ def remove_backup(self, backup_id, keep_physical=False):
             self.backups = _backups
 
 
-    def backup(self, defn, backup_id):
+    def backup(self, defn, backup_id, devices=[]):
         self.connect()
 
         self.log("backing up machine ‘{0}’ using id ‘{1}’".format(self.name, backup_id))
+        for d in devices:
+            self.log(" - {0}".format(d))
```

Comment:
It isn't needed and it can be deleted, I did that to follow the same behavior from the restore function. Thanks.
`def restore(self, defn, backup_id, devices=[]):
    self.stop()`
`    self.log("restoring machine ‘{0}’ to backup ‘{1}’".format(self.name, backup_id))
    for d in devices:
        self.log(" - {0}".format(d)`